server/chart/twitter/user-network.js

```diff
@@ -219,46 +226,43 @@ module.exports = {
             .then(() => ptools.delay(0))
         }
       }
-      promiseChain.then(()=> {
-        resolve(authorRelations);
-      })
-        .catch(function (reason) {
+      promiseChain
+        .then(()=> {
+          resolve(authorRelations);
+        })
+        .catch(reason => {
           debug(reason + " :continuing...");
           resolve(authorRelations);
         });
     });
   },
-  execute: function (filter, cb) {
+
+  execute(options, done) {
+    debug(options)
     if (twitter_scrape_method === 'twitter' && !twitterClient) {
-      cb(null, {message: "twitter client not ready"});
-      return;
+      return done(new Error('twitter client not ready'));
     }
 
     //check to see if we have already built this network
-    this.getEventNetwork(filter.eventid)
+    this.getEventNetwork(options.eventId)
       .then(network=>{
-        if(!network || network.status <= 0){
-          cb(null, "working");
+        if(!network || network.status <= 0) {
           if(!network) {
             const EventNetwork = app.models.EventNetwork;
-            EventNetwork.create({event_id: filter.eventid, status: 0, data: {}}, function (err,obj){
+            EventNetwork.create({event_id: options.eventId, status: 0, data: {}}, function (err,obj){
               network = obj;
```

Comment:
I think we do need to wrap this up to make sure network gets set...there is a race here but creation is so fast I think we are getting lucky.